src/crypto/tls/handshake_client_test.go

```diff
@@ -296,6 +296,7 @@ func (test *clientTest) run(t *testing.T, write bool) {
 	if config == nil {
 		config = testConfig
 	}
+	config.CipherSuites = allCipherSuites()[2:]
```

Comment:
Instead of doing this, we can regenerate the test vectors.

And we should add tests for ChaCha.